.github/workflows/release.yml

```
@@ -0,0 +1,70 @@
+name: Release Workflow
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)